packages/generator/tests/acceptance/test_plugin_generator.py

```diff
@@ -1,35 +1,36 @@
 import pathlib
+from functools import partial
 
 import pytest
-
-from ni_measurement_plugin_sdk_generator.plugin import create_measurement
+from click.testing import Result
 
 
 def test___command_line_args___create_measurement___render_without_error(
-    test_assets_directory: pathlib.Path, tmp_path_factory: pytest.TempPathFactory
+    create_measurement: partial[Result],
+    test_assets_directory: pathlib.Path,
+    tmp_path_factory: pytest.TempPathFactory,
 ) -> None:
     temp_directory = tmp_path_factory.mktemp("measurement_files")
     golden_path = test_assets_directory / "example_renders" / "measurement"
     filenames = ["measurement.py", "SampleMeasurement.serviceconfig", "start.bat", "_helpers.py"]
 
-    with pytest.raises(SystemExit) as exc_info:
-        create_measurement(
-            [
-                "Sample Measurement",
-                "--measurement-version",
-                "1.2.3.4",
-                "--ui-file",
-                "MeasurementUI.measui",
-                "--service-class",
-                "SampleMeasurement_Python",
-                "--description-url",
-                "https://www.example.com/SampleMeasurement.html",
-                "--directory-out",
-                temp_directory,
-            ]
-        )
+    result: Result = create_measurement(
```

Comment:
This type hint should not be needed. Type checkers should infer it based on the return type of `create_measurement`, and this isn't a documented attribute/constant.